### ISSUE TITLE
💻UI Improvement: live statistics dashboard and changing buttons positions. 

### ISSUE THREAD

**Issue by gisellandrade** [hedyorg/hedy#5424]
As this is related to Jesus issue's #4580 and we disscused, this how the dashboard would look like! Let me know if there's anything else to change. @jpelay  This is the first approach, even though I'm not sure on how to show students information :v 

![image](https://github.com/hedyorg/hedy/assets/154460618/ca8af584-7df8-43f0-9710-f75b012de73e)

![image](https://github.com/hedyorg/hedy/assets/154460618/82ca2ca1-f0bc-4bb0-9578-33d8e891e922)

**Comment by hasan-sh** [comment]
Hi, what's the difference between more options and the settings button? Also it'd be nice to change the position of the survey since it's a bit lonely right now! Perhaps add it as an icon near the class title? 

I have a question. Is this page the class-overview or the live statistic? Or both merged?!

**Comment by jpelay** [comment]
> I have a question. Is this page the class-overview or the live statistic? Or both merged?!

This is actually the /class page! We are merging all of the other pages into one with more functionality.

**Comment by gisellandrade** [comment]
Hi! Yes, those pages are going to be the same! That's what Jesus and I disscussed a while ago, but now we're including the gaphics and also i don't know if there's anything else you consider I should add? Btw, I just made the changes that @hasan-sh mentioned, if there's anything else let me know! 

![image](https://github.com/hedyorg/hedy/assets/154460618/480cf9bc-d0ac-4010-8e99-b2e93baf9f30)
![image](https://github.com/hedyorg/hedy/assets/154460618/c7cbbbba-8b8b-472d-bb48-25d26d898218)
